tools/coq_makefile.ml

```diff
@@ -203,7 +203,7 @@ let generate_conf_coq_config oc args bypass_API =
   section oc "Coq configuration.";
   let src_dirs = if bypass_API
                  then Coq_config.all_src_dirs
-                 else Coq_config.api_dirs @ Coq_config.plugins_dirs @ ["-open API"] in
+                 else Coq_config.api_dirs @ Coq_config.plugins_dirs in
```

Comment:
`src_dirs` is a list of directories, putting a flag for `ocamlc` there is not doing anything reasonable.